CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added `oe_sgx_get_signer_id_from_public_key()` function which helps a verifier of SGX
   reports extract the expected MRSIGNER value from the signer's public key PEM certificate.
+- OE SDK can now be built for Simulation Mode on non SGX x64 systems by passing HAS_QUOTE_PROVIDER=off.
+  This is enabled by dynamically loading SGX Enclave Common Library instead of linking it into
+  the host application.
```

Comment:
Can we reflect what end users will see as well?
  the host application. Users installing the SDK packages on non SGX X64 systems will be able to run the helloworld, switchless and fileencryptor samples in simulation mode.

Reply:
I have reworded it to indicate the user experience.